rules_building_block/privilege_escalation_expired_driver_loaded.toml

```diff
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2023/06/26"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/06/26"
+bypass_bbr_timing = true
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies an attempt to load a revoked or expired driver. Adversaries may bring outdated drivers with vulnerabilities
```

Comment:
The description indentation seems a bit off

Reply:
Indented at 120 🤔